upup/pkg/kutil/delete_cluster.go

```diff
@@ -1213,6 +1215,14 @@ func DeleteVPC(cloud fi.Cloud, r *ResourceTracker) error {
 	return nil
 }
 
+func DumpVPC(r *ResourceTracker) (interface{}, error) {
```

Comment:
That's right.  We should refactor discovery from deletion.

There's also another use case - right now if we switch e.g. from private -> public, we're likely to orphan a bunch of resources.  We have no way to delete them right now.  We should have some form of garbage collection.  Although I guess that is more similar to delete :-)